etl/app/index.js

```diff
@@ -21,17 +69,15 @@ app.on('ready', async () => {
   }
 
   log.info('Creating tables, running load, and scheduling load to run daily');
-
-  // Create and load new schema
-  await database.runJob(true);
+  etlJob();
 
   // Schedule ETL to run daily at 3AM
   cron.schedule(
     '0 3 * * *',
     async () => {
```

Comment:
We can drop this `async` keyword if we're not going to use `await` with the task call:
```suggestion
    () => {
```